### PR TITLE
Fixes visualization files

### LINKED ISSUE #157: Edit maximum value for y axis in visualization graphs
Values for residential building types are not showing up since the max y axis values are determined by commercial building types max values.

### LINKED ISSUE #158: Check units for Gas:Facility in visualization graphs
The Gas:Facility values are expressed in kBTU in the default csv reports. They are converted to kWh for visualization, the unti in the graph needs to reflect that.

### PR DESCRIPTION
**Pull request by tanushree04**
### Pull Request Description
- The unit for Gas:Facility should be updated to be kWh
- Reduce maximum y-axis value in annual end use graphs to show results for residential simulations better.

### Checklist (Delete lines that don't apply)

- [x] All ci tests pass (green)
- [x] An [issue](https://github.com/urbanopt/urbanopt-cli/issues) has been created (which will be used for the changelog)
- [x] This branch is up-to-date with develop

### CODE REVIEW

**Comment by tanushree04** [comment]
This [line](https://github.com/urbanopt/urbanopt-cli/pull/159/commits/1404621d26ce9868482792808fc71c3730019fb5#diff-0a74203d7c5c2ed62cb62466de1167126a4214c4130e973cd0acdeef9bb0f067R487) is setting the maximum sum of all end use values for the features as the y-axis of the end use graphs. Since this may be a high number, some features with lower end use totals are not showing up as clearly.

image

Is updating the total end use values for each feature and setting it as the y axis maxmium value individually a solution @kflemin? This does not allow comparing all feature end use results in the scale though.

File: example_files/visualization/input_visualization_feature.html [hunk 1]
```diff
@@ -467,13 +466,15 @@ <h2>Annual End Uses</h2>
...
+          var total_value_feature = 0
           // annual end use
           if(feature['complete_simulation'] == true){
           $scope.annualEndUseChartData[feature.name] = [];
           // Find all applicable end uses
           _.forEach(applicableEndUseKeys, function (endUseKey){
               var endUseValue = feature.annual_values[endUseKey]
-
+              
+              total_value_feature += endUseValue
```
Comment (tanushree04) [inline example_files/visualization/input_visualization_feature.html:477]: @kflemin I have added the `total_value_feature` as the local average for each feature

File: example_files/visualization/input_visualization_feature.html [hunk 2]
```diff
@@ -483,13 +484,17 @@ <h2>Annual End Uses</h2>
                 y: endUseValue
               }]
             });
+
           });
-          $scope.annualEndUseChartOptions.chart['yDomain'] = [0, _.round(1.1 * annualTotalYMax)];
-        
+          if (annualTotalYMax > total_value_feature + 20000000) {
```
Comment (tanushree04) [inline example_files/visualization/input_visualization_feature.html:489]: The difference between the sum of all end uses for a commercial and residential building was ~ 20,000,000 kWh.

File: example_files/visualization/input_visualization_feature.html [hunk 3]
```diff
@@ -483,13 +484,17 @@ <h2>Annual End Uses</h2>
...
+          $scope.annualEndUseChartOptions.chart['yDomain'] = [0, _.round(total_value_feature)];
```
Comment (tanushree04) [inline example_files/visualization/input_visualization_feature.html:490]: For some reason the $scope.annualEndUseChartOptions.chart['yDomain'] sets the yDomain for all graphs and doesn't update for each feature based on whehter the condition is satisfied or not.

**Comment by tanushree04** [comment]
The visualization tests are failing because the scenario_visualization class in the scenario gem is updated to take in 3 arguments in the uo_cli.rb but this change is in the develop branch of scenario gem and not in the released version that the cli is pointing too.

**Comment by tanushree04** [comment]
> The visualization tests are failing because the scenario_visualization class in the scenario gem is updated to take in 3 arguments in the uo_cli.rb but this change is in the develop branch of scenario gem and not in the released version that the cli is pointing too.

I can update the gemfile for now to point to the develop branch of scenario gem for the tests to pass.

**Comment by vtnate** [comment]
@tanushree04 This looks great! 🚀
In the `Annual End Uses` section, updating the end-use tags would be nice. It's a little confusing to see tags for end uses that aren't in the graph, and it would be super fancy if we could have the tag next to the color of the bar that it represents. Is that a reasonable request, or is that crazy hard?

**Comment by tanushree04** [comment]
@vtnate should we go ahead and merge this? I can add that as a feature enhancement request.

**Comment by vtnate** [comment]
> @vtnate should we go ahead and merge this? I can add that as a feature enhancement request.

Yeah, what you have here satisfies the Issues. I'll merge this now. Well done!

**Pull request merged**